EMF/common/religions/hip_religions.txt

```diff
@@ -19,8 +19,8 @@ hip_religion_group = {
     hip_religion = {
         graphical_culture = westerngfx
 
-        icon = 60
-        heresy_icon = 60
+        icon = 62
```

Comment:
This part I forgot about, however. Can't have Isis having the wrong religion icon.